web_src/js/features/common-global.js

```diff
@@ -142,6 +142,18 @@ export function initGlobalCommon() {
       window.location = href;
     }
   });
+
+  // loading-button this logic used to prevent push one form more than one time
+  $(document).on('click', '.button.loading-button', function (e) {
+    const $btn = $(this);
+
+    if ($btn.hasClass('loading')) {
+      e.preventDefault();
+      return false;
+    }
+
+    $btn.addClass('loading disabled');
```

Comment:
```suggestion
    $btn.addClass('loading disabled');
    $btn.prop('disabled', true);
```

Should we use `$btn.prop('disabled', true);` to disable the button entirely? Then it's not clickable anymore.

Reply:
looks semantic ui has handled it, just add `disabled` class is enough.

Reply:
But what if it is not a Fomantic UI button? For example, what if some one writes a form without `.ui` class but uses your code?

If you require the developers to use your code in Fomantic UI context, you should change the selector to `.ui.button .xxxxxx` to make sure the code only works for Fomantic UI.